source/default/_patterns/02-molecules/vue-widget/__tests__/vue-clock.test.js

```diff
@@ -20,14 +23,15 @@ describe('vue-clock.vue', () => {
     expect(hoursHand.style.transform).toContain('rotate');
   });
 
-  it('updates the seconds hand', done => {
+  it('updates the seconds hand', () => {
     const { getByTestId } = render(vueClock);
     const secondsHand = getByTestId('seconds-hand');
     const before = secondsHand.style.transform;
+
     setTimeout(() => {
       const after = secondsHand.style.transform;
       expect(before).not.toEqual(after);
-      done();
+      // done();
```

Comment:
I'm guessing the afterEach(cleanup) removes this?

Reply:
The `useFakeTimers()` removes the need for this.